apisix/plugins/fault-injection.lua

```diff
@@ -15,13 +15,66 @@
 -- limitations under the License.
 --
 local core = require("apisix.core")
+local expr = require("resty.expr.v1")
 
 local sleep = core.sleep
 local random = math.random
+local ipairs = ipairs
 
 local plugin_name   = "fault-injection"
 
 
+local vars_schema = {
```

Comment:
I think we need to remove the `vars_schema`.
1. we can do the check better with `expr.new` when checking the schema.
2. when `not`/`or` is supported directly in the expr, this schema is outdated.
3. this schema contains many incorrect hardcoded limitations, like the `maxLength` of operator is `2` (not enough for `has`), and an expression may contain 4 elements if `!` is used, etc.

Reply:
updated.

Reply:
No, you don't remove it.

Reply:
I think just keep:
```
local vars_schema = {
    type = "array",
    maxItems = 20
}
```
is enough. The remain can be validated via `expr.new`.

Reply:
Okay, let me try it.

Reply:
updated.